pkgs/top-level/all-packages.nix

```diff
@@ -24307,6 +24307,8 @@ with pkgs;
 
   newt = callPackage ../development/libraries/newt { python = python3; };
 
+  nfft = callPackage ../development/libraries/nfft { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format to avoid having to edit this file